app/data/generated/ivy-client.ts

```diff
@@ -4,6 +4,7 @@
  * Axon Ivy OpenAPI
  */
 import { customFetch } from '../custom-fetch';
+import type { Form } from '../form-api';
```

Comment:
this is a cycle, `Form` points to `HdBean` which is part of this ivy-client.ts

note that this client file gets generated based on the openapi spec. If you have merged the new rest endpoint:
- run https://jenkins.ivyteam.io/job/core_openapi/

and then:
npm run download:openapi
npm run codegen:openapi